packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -116,48 +78,18 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
    * Creates a Model Instance Document.
    * @param ceramic - Instance of CeramicAPI used to communicate with the Ceramic network
    * @param content - Genesis contents. If 'null', then no signature is required to make the genesis commit
-   * @param metadata - Genesis metadata
+   * @param metadata - Genesis metadata, including the model that this document belongs to
    * @param opts - Additional options
    */
   static async create<T>(
     ceramic: CeramicApi,
-    content: T | null | undefined,
-    metadata?: ModelInstanceDocumentMetadataArgs,
+    content: T | null,
+    metadata: ModelInstanceDocumentMetadata,
```

Comment:
metadata is now required, and must have a `model` field within it.  I debated pulling model up out of `metadata` into a top-level required positional arg, but then we'd either need an optional `controller` field or an optional `metadata` field with only a single `controller` property.  Keeping all the metadata in a `metadata` field makes the API a bit more extensible so that's what I went with.  But I could easily be convinced to pull model out to the top level